### ISSUE TITLE
Native audio upload feature

### ISSUE THREAD

**Issue by SpriterGors** [bluesky-social/social-app#5869]
### Describe the Feature

For musicians like me, showing our works is crucial, however right now we need to link our stuff elsewhere, or upload an entire video just for audio. This is not optimal as it ends up consuming more bandwidth than a simple audio file would require, for something that cannot even be 1 minute long.

### Attachments

i think that a "simple" way to implement it would be just adding an audio widget like this (probably the length should extend to the entirety of the post):
![image](https://github.com/user-attachments/assets/8c1a93fa-18ae-4260-a691-77c2ee29c009)

Optional features would be for it to display the waveform and the filename, or other metadata present in the file.

### Additional Context

brought up by this post https://bsky.app/profile/bjoernmeansbear.bsky.social/post/3l6wv44cckx2y

**Comment by Feldstrom** [comment]
As a musician, Im begging for this functionality!

**Comment by zecul** [comment]
As a record shop, I'm begging for this feature too.

**Comment by Tamschi** [comment]
This shouldn't be a blocker for the overall feature, but Mastodon has a very nifty floating media player that detaches from the post when it scrolls out of view. Pretty such it also makes it easy to jump back to or open the post by tapping on it, and on Android it integrates with the system-wide playback controls (at least in the official app, but maybe also via browser?).

Something like that would make longer audio/music very enjoyable to find in the app. While floating, the player also shouldn't auto-dismiss when playback ends, so that the user can still quickly open the post to interact with it.

**Comment by rgeraldporter** [comment]
+1 to this because as a podcaster and audiobook creator this is crucial for promotion. Making videos just to post audio is so much extra unnecessary work, and I'm sure its also adding unnecessarily to the bandwidth and storage costs.

**Comment by yankscally** [comment]
I will be trying my best to make this happen.

This makes so much sense. 

Musicians like us need this feature.

Begging!

**Comment by persistensk** [comment]
Yes, please! I’d like to post my recordings of  bird songs to [hashtag]birdsongs. Trying to make these audio files
 into videos would be time-consuming. Also, using video to share just audio would be wasteful. Audio files are much more compact and use less energy to store and transmit. In addition, the ability to share audio files would increase accessibility.  Creators who are blind or have visual disabilities could post voice content, without having to make videos. Thanks in advance!

**Comment by HologramOfMe** [comment]
I am joining the chorus. I would love a simple way to share audio clips and sounds I've designed and created.

**Comment by ioletsgo** [comment]
The way Mastodon (and forks like glitch-soc) handles audio uploading is by adding an image-size audio widget.

<img width="687" height="820" alt="Image" src="https://github.com/user-attachments/assets/c2f61eca-4088-4aa0-8b3b-f9ea4a5de5c6" />

**Comment by lethargistic** [comment]
bump

**Comment by luan-u** [comment]
> I am joining the chorus. I would love a simple way to share audio clips and sounds I've designed and created.

Although not _really_ simple, a workaround is to create a video with a static image and the desired audio.

You could pick an image (or create one) to convey the content is audio only. like this:
![Image](https://github.com/user-attachments/assets/22326ce8-4801-42b1-914a-cbc578a2759e)

**Comment by SylvieDotCat** [comment]
Another +1 for  this 

I've recently been getting into music and not having a native audio upload feature is really a killer for those who're in this field. Understand that yes we can include links to Soundcloud/YT/etc but making a direct easy upload for Music-Audio based users would reallly help in the music community feeling more welcomed on the site.

**Comment by IsaacMarovitz** [comment]
Would very much like to have this feature too

**Comment by SpriterGors** [comment]
> > I am joining the chorus. I would love a simple way to share audio clips and sounds I've designed and created.
> 
> Although not _really_ simple, a workaround is to create a video with a static image and the desired audio.
> 
> You could pick an image (or create one) to convey the content is audio only. like this: ![Image](https://github.com/user-attachments/assets/22326ce8-4801-42b1-914a-cbc578a2759e)

Thinking about this implementation,

We know that currently, Bluesky App does not really have support for animated GIFs, with the workaround being automatic conversion of GIF to MP4 in the same UI.

What if the solution is for Bluesky to be able to output a MP4 file with image + audio file input? Would this be feasible?
